File: all/abc267/abc267_a/abc267_a.java
import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner stdIn = new Scanner(System.in);

    String[] ptn = {"Monday", "Tuesday", "Wednesday", "Thursday", "Friday"};
    String s = stdIn.next();

    for(int i = 0; i < 5; i++) {
      if(s.equals(ptn[i])) {
        System.out.println(5 - i);
        break;
      }
    }
  }
}

File: all/abc267/abc267_a/abc267_a.cs
class A
{
  static void Main()
  {
    string s = Console.ReadLine();
    string[] ptn = {"Monday", "Tuesday", "Wednesday", "Thursday", "Friday"};

    for(int i = 0; i < 5; i++)
    {
      if(s == ptn[i])
      {
        Console.Write(5 - i);
        return;
      }
    }
  }
}
